addons/test_mail/models/test_mail_models.py

```diff
@@ -112,7 +112,16 @@ class MailTestStandard(models.Model):
     user_id = fields.Many2one('res.users', 'Responsible', tracking=True)
     container_id = fields.Many2one('mail.test.container', tracking=True)
     company_id = fields.Many2one('res.company')
+    track_always_hide_tracking = fields.Boolean()
+    track_message = fields.Char(default='')
 
+    def _track_filter_displayed_tracking_values(self, tracking_values):
+        if self.track_always_hide_tracking:
+            return self.env['mail.tracking.value']
+        return super()._track_filter_displayed_tracking_values(tracking_values)
+
+    def _track_get_message(self, changes):
+        return self.track_message
```

Comment:
Maybe we could have something based on changes (like displaying changes), just to have a more dynamic test ?